README.md

```diff
@@ -1,3 +1,59 @@
 # KubeSaw Docs
 
 This repository contains the documentation website code and Markdown source files for kubesaw.github.io.
+
+## Contributing
+
+The website is built using [mkdocs](https://www.mkdocs.org/getting-started/#getting-started-with-mkdocs) 
+
+### Prerequisites
+
+- python3
+```shell
+python3 --version
+Python 3.11.5
+```
+
+### 1.  Create an isolated virtual environment
```

Comment:
Should we mention that the contributor should first fork the repo (and clone it)?

Reply:
Good point, I usually give that for grant but better to be explicit ! Thanks!